_resources/port1.0/group/qmake5-1.0.tcl

```diff
@@ -57,7 +57,8 @@ pre-configure {
     #
     # avoid --show-sdk-path since it is not available on all platforms
     # see https://github.com/macports/macports-ports/commit/9887e90d69f4265f9056cddc45e41551d7400235#commitcomment-49824261
-    if {[catch {exec -ignorestderr /usr/bin/xcrun --sdk macosx${configure.sdk_version} --find ld} result]} {
+    if {[catch {exec -ignorestderr /usr/bin/xcrun --sdk macosx${configure.sdk_version} --find ld 2>@1 > /dev/null} result]} {
```

Comment:
The `result` could also be removed since it is unused.

Reply:
done, thx